pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "arches-her"
+readme = "README.md"
+authors = [
+    {name = "Getty Conservation Institute", email = "contact@archesproject.org"}
+]
+license = {text = "GNU AGPL3"}
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
@chiatt - should this be Beta?

Reply:
Would you agree we should be using `5 - Production/Stable`?

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers

Reply:
We will do that in the stable/1.0.0 branch once this is merged in.

Reply:
We can just leave this in beta so it is ready for patch fixes as soon as stable/1.0.0 is created.

Reply:
Sounds good!